packages/api/models/granules.js

```diff
@@ -673,9 +636,31 @@ class Granule extends Manager {
    * @returns {Promise}
    */
   async storeGranulesFromCumulusMessage(cumulusMessage) {
-    const granuleRecords = await this.constructor
-      ._getGranuleRecordsFromCumulusMessage(cumulusMessage);
-    return Promise.all(granuleRecords.map(this._validateAndStoreGranuleRecord, this));
+    const granules = getMessageGranules(cumulusMessage);
```

Comment:
See comments on getMessageGranules from sf-even-sqs-to-db-records

Reply:
0fdc08fbe7c304be001b7374a8a24b5fe445eaab